app/top-page.module.css

```
@@ -0,0 +1,355 @@
+}
+.text {
+  font-size: 1.3rem;
+  font-family: Yu Mincho;
+  color: rgb(0, 0, 0);
+  text-align: left;
+  margin-top: 0px;
+}
+.footer {
+  margin-top: 60px;
+  background:linear-gradient(150deg, #0b69eb 70%, #ffffff 100%);
+  padding: 20px;
```

Stylelint (17.9.0)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)